### PR TITLE
Typo fix ("namespaced" > "namespace")

### PR DESCRIPTION
**Pull request by bobbypower-asm**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [X] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Review by ericvn — APPROVED** [review]

File: content/en/docs/reference/glossary/namespace-sameness.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ test: n/a
 
 Within a multicluster mesh, [namespace sameness](https://github.com/kubernetes/community/blob/master/sig-multicluster/namespace-sameness-position-statement.md)
 applies and all namespaces with a given name are considered to be the same namespace. If multiple clusters contain a
-`Service` with the same namespaced name, they will be recognized as a single combined service. By default, traffic is
+`Service` with the same namespace name, they will be recognized as a single combined service. By default, traffic is
```
Comment (frankbu) [inline content/en/docs/reference/glossary/namespace-sameness.md:8]: I don't think this change is correct. The service has to have the same namespace and name, which is what "same namespaced name" means. By saying the "same namespace name" it's just saying the name of the namespaces have to be the same, nothing about the service name itself.

Comment (bobbypower-asm) [reply content/en/docs/reference/glossary/namespace-sameness.md:8]: Understood! I will revert this change now.

Comment (ericvn) [reply content/en/docs/reference/glossary/namespace-sameness.md:8]: My thought was that the service would already have the same same so implying same namespace means both are equal. I'm not sure that namespaced really means "have the same namespace and name", although in this case that's how it was used. 

I would entertain that we extend this to say the same service name and namespace to be clear.

Comment (frankbu) [reply content/en/docs/reference/glossary/namespace-sameness.md:8]: I agree, let's make it unambiguous by saying `the same service name and namespace`.

**Pull request merged**